rasa/engine/runner/dask.py

```diff
@@ -100,6 +100,10 @@ def run(
         try:
             dask_result = dask.get(run_graph, run_targets)
             return dict(dask_result)
+        except KeyError as e:
+            raise GraphRunError(
+                f"Could not find key {e} in the graph. Error running runner."
```

Comment:
Should we expand this message to a more descriptive like:
```python
f"Could not find key {e} in the graph. Error running runner.",
f"Please check that you are running bot developed with CALM instead of bot developed with previous version of dialog management (DM1)."
```
?